dev/Code/CryEngine/RenderDll/Common/RenderThread.cpp

```diff
@@ -3355,7 +3358,7 @@ void SRenderThread::FlushAndWait()
 // Flush current frame without waiting (should be called from main thread)
 void SRenderThread::SyncMainWithRender()
 {
-    AZ_PROFILE_FUNCTION_STALL(AZ::Debug::ProfileCategory::Renderer);
+    AZ_PROFILE_FUNCTION(AZ::Debug::ProfileCategory::Renderer);
```

Comment:
This is not a stall function. It handles the buffer swaps and render thread submission.